sdk/keyvault/azure-security-keyvault/azure/security/keyvault/certificates/_client.py

```diff
@@ -0,0 +1,222 @@
+        # type: (str, Optional[CertificatePolicy], Optional[bool], Optional[datetime], Optional[datetime], Optional[Dict[str, str]], Mapping[str, Mapping[str, Any]]) -> CertificateOperation
+        pass
+
+    def get_certificate(self, name, version=None, **kwargs):
+        # type: (str, Optional[str]) -> Certificate
+        pass
+
+    def delete_certificate(self, name, **kwargs):
+        # type: (str) -> DeletedCertificate
+        pass
+
+    def get_deleted_certificate(self, name, **kwargs):
+        # type: (str) -> DeletedCertificate
+        pass
+
+    def purge_deleted_certificate(self, name, **kwargs):
+        # type: (str) -> None
+        pass
+
+    def recover_deleted_certificate(self, name, **kwargs):
+        # type: (str) -> Certificate
+        pass
+
+    def import_certificate(
+        self,
+        name,
+        base64_encoded_certificate,
+        password=None,
+        policy=None,
+        enabled=None,
+        not_before=None,
+        expires=None,
+        tags=None,
+        **kwargs
+    ):
+        # type: (str, str, Optional[str], Optional[Policy], Optional[bool],Optional[datetime], Optional[datetime], Optional[Dict[str, str]], Mapping[str, Any]) -> Certificate
+        pass
+
+    def get_certificate_policy(self, name, **kwargs):
+        # type: (str) -> CertificatePolicy
+        pass
+
+    def update_certificate_policy(self, name, policy, enabled=None, not_before=None, expires=None, tags=None, **kwargs):
+        # type: (str, CertificatePolicy) -> CertificatePolicy
+        pass
+
+    def update_certificate(self, name, version, not_before=None, expires=None, enabled=None, tags=None, **kwargs):
+        # type: (str, str, Optional[bool], Optional[Dict[str, str]]) -> Certificate
+        pass
+
+    def merge_certificate(
+        self, name, x509_certificates, enabled=True, not_before=None, expires=None, tags=None, **kwargs
+    ):
+        # type: (str, list[str], Optional[bool], Optional[Dict[str, str]]) -> Certificate
+        pass
+
+    def backup_certificate(self, name, **kwargs):
+        # type: (str) -> bytes
+        pass
+
+    def restore_certificate(self, backup, **kwargs):
+        # type: (bytes) -> Certificate
+        pass
+
+    def list_deleted_certificates(self, include_pending=None, **kwargs):
+        # type: (Optional[bool]) -> Iterable[DeletedCertificate]
+        pass
+
+    def list_certificates(self, include_pending=None, **kwargs):
+        # type: (Optional[bool]) -> Iterable[CertificateBase]
+        pass
+
+    def list_certificate_versions(self, name, **kwargs):
+        # type: (str) -> Iterable[CertificateBase]
+        pass
+
+    def create_contacts(self, contacts, **kwargs):
+        # type: (Iterable[Contact]) -> Iterable[Contact]
+        # TODO: rest api includes an id should it be returned?
+        pass
+
+    def list_contacts(self, **kwargs):
+        # type: () -> Iterable[Contact]
+        pass
+
+    def delete_contacts(self, **kwargs):
```

Comment:
Will this make a single REST call or multiple ones?

Reply:
it would be making a single REST call to delete all contacts in the specified keyvault.